client/landing/stepper/hooks/use-is-site-big-sky-eligible.ts

```diff
@@ -16,15 +17,30 @@ export function useIsBigSkyEligible(): boolean | null {
 		( select ) => ( select( ONBOARD_STORE ) as OnboardSelect ).getGoals(),
 		[ site ]
 	);
-	const validGoals = [ 'other', 'promote' ];
 
-	const configEnabled = config.isEnabled( 'calypso/ai-assembler' );
-	if ( ! configEnabled ) {
+	const featureFlagEnabled = config.isEnabled( 'calypso/big-sky' );
+	// For testing purposes. Will bypass the ExPlat experiment results.
+	const bypassExperiment = config.isEnabled( 'big-sky' );
+	const [ isLoadingExperiment, experimentAssignment ] = useExperiment(
+		'calypso_signup_onboarding_bigsky_soft_launch',
+		{
+			isEligible: featureFlagEnabled && ! bypassExperiment,
+		}
+	);
+
+	if ( ! featureFlagEnabled ) {
 		return false;
 	}
 
+	const variantName = experimentAssignment?.variationName;
+	const isInBigSkyExperiment = ! isLoadingExperiment && variantName === 'treatment';
```

Comment:
I think checking for loading doesn't add value here.

```suggestion
	const isInBigSkyExperiment = variantName === 'treatment';
```

Reply:
addressed in [df014aa](https://github.com/Automattic/wp-calypso/pull/91781/commits/df014aab13f0cf29928db9e8394642d45e54ba4e)